### ISSUE TITLE
Vivliostyle crashes when using CSS namespaces

### ISSUE THREAD

**Issue by hlandau** [vivliostyle/vivliostyle.js#1172]
**Describe the bug**

Minimal reproducer:

```shell
$ ./node_modules/.bin/vivliostyle build test.xhtml
reusing global emitter
✖ TypeError: Cannot read properties of undefined (reading 'box-decoration-break') 
 TypeError: Cannot read properties of undefined (reading 'box-decoration-break')
    at Vi (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:321743)
    at u.calculateClonedPaddingBorder (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:588385)
    at u.saveEdgeBreakPosition (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:600950)
    at u.checkOverflowAndSaveEdgeAndBreakPosition (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:592361)
    at file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:594242
    at file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:230632
    at n (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:230338)
    at e.loop (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:230449)
    at e.loopWithFrame (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:230558)
    at u.skipEdges (file:///home/hugo/Downloads/xoq1/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:594038)
	---- async ---
	loop
	Frame.loop
	skipEdges
	layoutNext
	loop
	Frame.loop
	doLayout
	DefaultLayoutMode.doLayout
	AbstractLayoutRetryer.tryLayout
	layout
	loop
	Frame.loop
	layoutColumn
	loop
	Frame.loop
	createAndLayoutColumn
	loop
	Frame.loop
	layoutFlowColumns
	AsyncResult.thenAsync
	AsyncResult.thenAsync
	layoutContainer.inner
	layoutContainer
	AsyncResult.thenAsync
	Frame.loop
	layoutContainer
	loop
	Frame.loop
	layoutNextPage
	renderSinglePage
	renderPage
	loop
	Frame.loop
	renderPagesUpto
	resize
$ ./node_modules/.bin/vivliostyle --version
cli: 7.2.5
core: 2.24.3
$ cat test.xhtml
<?xml version="1.0" encoding="UTF-8"?>
<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:d="https://www.devever.net/ns/dedoc" lang="en" xml:lang="en">
  <h:head>
    <h:style type="text/css"><![CDATA[
@namespace url(http://www.w3.org/1999/xhtml);
@namespace d url(https://www.devever.net/ns/dedoc);
d|p { display: block; }
    ]]></h:style>
  </h:head>
  <h:body>
    <doc xmlns="https://www.devever.net/ns/dedoc" xml:lang="en">
      <p>
        <h:p>Foo.</h:p>
      </p>
      <p>
        <h:p>Bar.</h:p>
      </p>
    </doc>
  </h:body>
</h:html>
```

**Comment by MurakamiShinyu** [comment]
@hlandau Thanks for reporting bug. Fixed in PR #1179.

**Comment by MurakamiShinyu** [comment]
## Test

Test file: https://gist.githubusercontent.com/MurakamiShinyu/c731983d57314176cbcf84badd6cd689/raw/8f40eac3340703c2fc44b107d5c913aef9400770/issue1172-test.xhtml

Test with Vivliostyle Viewer:
- Before this fix (v2.24.3): https://vivliostyle.github.io/viewer/v2.24.3/#src=https://gist.githubusercontent.com/MurakamiShinyu/c731983d57314176cbcf84badd6cd689/raw/8f40eac3340703c2fc44b107d5c913aef9400770/issue1172-test.xhtml
- After this fix: https://vivliostyle.vercel.app/#src=https://gist.githubusercontent.com/MurakamiShinyu/c731983d57314176cbcf84badd6cd689/raw/8f40eac3340703c2fc44b107d5c913aef9400770/issue1172-test.xhtml
